Fixed some Display Abilities

Uveso opened the pull request:
While working on the new automatic-ability function, i found wrong Abilities inside some units.

Uveso committed:
Fixed some Abilities

aeoncleanse commented on BlackOpsUnleashed/units/BAA0401/BAA0401_unit.bp:
```diff
@@ -100,9 +100,6 @@ UnitBlueprint {
     Display = {
-        Abilities = {
-            'Orbital',
-        },
```
Is this not orbital?

aeoncleanse commented on BlackOpsUnleashed/units/BAB1106/BAB1106_unit.bp:
```diff
@@ -53,9 +53,6 @@ UnitBlueprint {
     Display = {
-        Abilities = {
-            'Storage',
-        },
```
Storage can probably storage?

aeoncleanse commented:
Looks like there could be some issues here. Highlighted a couple. I think all abilities _sshould_ be using the <LOC ... syntax mind, so we get proper translations.

Uveso commented:
Yes i added the `<LOC ability_xyz>` to all abilities, so we can translate them.
These are all abilities from BlackOps who need translation:

```
ability_groundattack="ability_groundattack"
ability_anti="Anti-Teleport Field:Prevents Teleportation"
ability_gat="Gattling Pulse Laser"
ability_torpturn="Torpedo Redirection System"
ability_shieldisrupt="Shield Disruptor"
ability_antiteleport="Anti-Teleport Field: Prevents Teleportation"
ability_ran="Random Weapon:"
ability_directfire="ability_directfire"
ability_attkdrones="Anti-Air Drones"
ability_energy="High Energy Consumption"
ability_quanRes="Quantum Power"
ability_tar="Forced Targeting"
ability_indirectfire="ability_indirectfire"
ability_lambdafield="Lambda Field"
ability_cloakfield="Cloak Field"
ability_flm="Flamethrower"
ability_teleport="Teleport"
ability_mine="Deploys Land Mines"
ability_lsr="Beam Laser"
ability_rkt="Rocket-propelled Grenades"
```

I deleted the orbital and storage abilitiy because the tooltip already has this information.
Its like engineers with repair or reclaim.

Will be clear if you see the tooltips:
![toolabi1](https://cloud.githubusercontent.com/assets/17804547/17624805/7385c570-60a6-11e6-8a4f-421f905b50c8.png)
![toolabi2](https://cloud.githubusercontent.com/assets/17804547/17624808/77c6de9e-60a6-11e6-86e0-376366f4ada1.png)

aeoncleanse commented on BlackOpsUnleashed/units/BEL0308/BEL0308_unit.bp:
```diff
@@ -64,7 +64,7 @@ UnitBlueprint {
     Display = {
         Abilities = {
-            'Directfire',
+            'ability_directfire',
```
Not using the LOC. Just one example.

aeoncleanse commented:
The abilities display is also shown in other places where that full explanation window isn't visible. It's probably best to keep them. If that means it gets said twice, remove it from the other location.

Uveso commented:
closed,
next attempt can be found here :  #153
